src/Sulu/Bundle/WebsiteBundle/Resources/public/js/validation/custom.js

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of the Husky Validation.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ *
+ */
+
+define([
+    'type/default'
+], function(Default) {
+
+    'use strict';
+
+    return function($el, options) {
+        var defaults = {
+                instanceName: null
+            },
+
+            subType = {
+                setValue: function(content) {
+                    if (content === null) {
+                        content = {};
+                    }
+
+                    if (!content.hasOwnProperty('value') || !content.hasOwnProperty('position')) {
```

Comment:
is this check needed? if yes why no set the position to null when checking for it?